### DISCUSSION TITLE
Is there any event specifically designed for schema updates?

### DISCUSSION THREAD

**Discussion by gintarasgraciovas** [clientIO/joint#2593] [Q&A] [answered]
### Introduction

I'd like to track when schema updates occur so that I can send requests to the backend accordingly. Currently, there isn't a specific mechanism for this (that I'd know about), so I have to handle it in the following manner:
![image](https://github.com/clientIO/joint/assets/146833191/ed02ad4b-73d4-421e-9cfc-2bc1b77615ad)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
If you are _JointJS+_ user, the easiest way to detect graph changes is to use the `commandManager` plugin.

```ts
commandManager.on('stack', () => {
   sendToBackend(graph.toJSON());
});
```

**Reply by gintarasgraciovas** [reply]
Is it tracking adding new attributes to cell though?

**Reply by kumilingus** [reply]
By default, it's tracking all the changes made to the graph.

**Reply by gintarasgraciovas** [reply]
Thank you :)
